app/schema.py

```diff
@@ -130,8 +129,6 @@ class Config:
 
 
 class MeetingUpdate(BaseModel):
-    meeting_id: Optional[constr(max_length=255)]
-    created_at: Optional[datetime]
```

Comment:
Good preventative measures to control changes to the database while updating. This looks really good.